01_Core/week02/submissions/javascript.js

```diff
@@ -0,0 +1,16 @@
+var names = ["John Doe", "Mike Smith", "Gary Thompson", "Thomas Williams", "Joe Miller"];
+console.log(names.sort());
+/*
+document.getElementById("dos").innerHTML = names;
```

Comment:
if you are trying to add this list of names, rather than trying to insert the array as a block, you'll need to loop through each item, add it to a string and insert the string. Or you could use the join() method.